src/Cache/__tests__/Cache-test.js

```diff
@@ -266,11 +268,10 @@ describe('Cache', () => {
         Promise.resolve('baz value')
       );
 
-      // jest has some trouble with promises and timeouts within promises :(
-      jest.runAllTimers();
-      jest.runAllTimers();
-
-      expect(fs.writeFile).toBeCalled();
+      setTimeout(() => {
```

Comment:
uh, this is not really great and the `useRealTimers` isn't either. Is the problem here using native promises and the micro-task-queue? That does kinda suck, I feel like Jest should just swap out the native promise by default...

This test isn't really going to work because jest doesn't wait after a test, so you'd at least need to use fake timers and call runAllTimers after this setTimeout call. I'd much rather replace promises for the purpose of this test and advance the timers.